src/main/resources/templates/dean/fragments/user_table.html

```diff
@@ -29,7 +29,8 @@ <h4 class="card-title" id="from-actions-top-left">Created Users</h4>
                     <tr>
                         <td th:text="${user.name}"></td>
                         <td th:text="${user.userId}"></td>
-                        <td th:text="${user.createdAt != null ? user.createdAt : user.updatedAt }"></td>
+                        <td th:text="${user.createdAt != null ?
```

Comment:
This means that if `createdAt` is not null, display `createdAt`. Else display `updatedAt`. Shouldn't it be the other way round?

Reply:
We want to always show createdAt, but for extreme cases it may be null, so that's why updatedAt

Reply:
I think `${#temporals.format(user.createdAt != null ? user.createdAt : user.updatedAt, 'dd/MMM/yyyy HH:mm') }`

would be shorter and less repetitive

Reply:
Is there a way to set a default format for temporals ?

Reply:
Hmm, don't think so, they have some preset fomrats but that's it